src/shared/tpm2-util.c

```diff
@@ -2333,3 +2350,20 @@ int pcr_mask_to_string(uint32_t mask, char **ret) {
         *ret = TAKE_PTR(buf);
         return 0;
 }
+
+void tpm2_util_pbkdf(const void *key,
+                    size_t key_size,
+                    const void *input,
+                    size_t input_size,
+                    int iterations,
```

Comment:
given this cannot really be negative sounds weird accepting negative here. maybe just make this unsigned?

Reply:
Yep was just keeping it the same as openssl for no particular reason.